src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -267,20 +271,23 @@ async def pylint_lint(
 
     # We batch targets by their interpreter constraints to ensure, for example, that all Python 2
     # targets run together and all Python 3 targets run together.
+    # Note that Pylint uses the AST of the interpreter that runs it. So, we include any plugin
```

Comment:
```suggestion
    # Note that Pylint uses the `ast` module of the interpreter that runs it. So, we include any plugin
```
optional

Reply:
See https://github.com/pantsbuild/pants/pull/10817#discussion_r492468870. It uses Astroids, actually.

Reply:
Oh great!

Reply:
Ok, I still want to leave a comment here, but can't figure out a good one. I think this is fine.